tests/python/gaia-ui-tests/gaiatest/apps/base.py

```diff
@@ -19,8 +19,9 @@ def __init__(self, marionette):
         self.apps = GaiaApps(self.marionette)
         self.accessibility = Accessibility(self.marionette)
         self.frame = None
-        self.manifest_url = hasattr(self, 'manifest_url') and self.manifest_url or None
+        self.manifest_url_app = None
         self.entry_point = hasattr(self, 'entry_point') and self.entry_point or None
+        DEFAULT_APP_HOSTNAME = 'gaiamobile.org'
```

Comment:
Sorry, I think I wasn't clear enough in my explanation, on IRC. 

``` python
class MyClass:
    MY_CLASS_VARIABLE = 'foo' # This variable is accessible with self.

    def __init__(self):
        MY_LOCAL_VARIABLE = 'bar' # This one is not reachable with self
```

Martijn is right, we should move that variable before `__init__`